src/main/java/pix/commands/AddEvent.java

```diff
@@ -0,0 +1,37 @@
+package pix.commands;
+
+import pix.data.MyData;
+import pix.tasks.Event;
+import pix.ui.Ui;
+
+/**
+ * AddEvent class to add event to list of tasks.
+ */
+public class AddEvent extends Command {

```

Comment:
Would naming classes representing different types of tasks and event as "AddSomething" lower the class dealing scopes? E.g In the future, the class may need to encapsulate some other information about the task, the class name may sound confused in that case since it suppose to use as adding event/task only.